roles/translator/src/upstream_sv2/upstream.rs

```diff
@@ -436,6 +456,45 @@ impl Upstream {
         todo!()
     }
 
+    // async fn handle_update_hashrate(self_: Arc<Mutex<Self>>) -> ProxyResult<'static, ()> {
```

Comment:
Do we need this block? If so can you comment in code why we want to keep it here?